velox/expression/tests/ExpressionVerifier.cpp

```diff
@@ -144,9 +144,46 @@ std::vector<fuzzer::ResultOrError> ExpressionVerifier::verify(
   std::vector<fuzzer::ResultOrError> results;
   // Share ExpressionSet between consecutive iterations to simulate its usage in
   // FilterProject.
-  exec::ExprSet exprSetCommon(
-      plans, execCtx_, !options_.disableConstantFolding);
   exec::ExprSetSimplified exprSetSimplified(plans, execCtx_);
+  std::shared_ptr<exec::ExprSet> exprSetCommon;
```

Comment:
Hi @rui-mo, thank you for helping fix this in the fuzzer! Would it work if we just move the `exec::ExprSet exprSetCommon(plans, execCtx_, !options_.disableConstantFolding);` into the try-block at the original line 195?

Reply:
Hi @kagamiori, thanks for helping review! I notice the below comment and wonder if keeping ExprSet out of the for loop is intentional. Could you provide further advice here? Thank you.

> // Share ExpressionSet between consecutive iterations to simulate its usage in
  // FilterProject.

Reply:
You're right. We need to keep the creation of ExprSet out of the loop. Then could you move the logic of creating ExprSet into a helper method, to be something like `ExprSet exprSetCommon = createExprSetCommon(...)`?

Reply:
@kagamiori Updated following your suggestion. Could you help take another look? Thanks.